resources/analysisTools/snvPipeline/snvAnnotation.sh

```diff
@@ -17,7 +17,7 @@ declare -r outputDirectory=`dirname ${FILENAME_VCF_OUT}`
 relinked=false
 # (Re-)link bam file and index. The pipeline cannot handle i.e. .mdup.bam and .mdup.bai
 # Look up the index with .bam.bai. If the file exists nothing happens.
-if [[ ! -f ${TUMOR_BAMFILE_FULLPATH_BP}.bai ]]
+if [[ ! -f ${TUMOR_BAMFILE_FULLPATH_BP}.bai && ! -f ${TUMOR_BAMFILE_FULLPATH_BP}.crai ]]
```

Comment:
This block works for the old style of naming `.bai` files. Since we have adapted the current style of naming for CRAM files. I do not think we would need this block for CRAM files.

Reply:
> Since we have adapted the current style of naming for CRAM files.

Do you mean that "we" (who?) agreed to use `.bai` as suffix for CRAM index files?